test/cmdlineTests.sh

```diff
@@ -378,6 +379,8 @@ SOLTMPDIR=$(mktemp -d)
         then
             opts+=(-o)
         fi
+
+        sed -i.bak -E -e 's/pragma[[:space:]]+solidity[[:space:]]*(\^|>=)[[:space:]]*'"$developmentVersion"'/pragma solidity >0.0.1/' "$f"
```

Comment:
The verbosity of this one makes it less readable so I'd add a comment:
```suggestion
        # Disable the version pragma in code snippets that only work with the current development version.
        # It's necessary because x.y.z won't match `^x.y.z` or `>=x.y.z` pragmas until it's officially released.
        sed -i.bak -E -e 's/pragma[[:space:]]+solidity[[:space:]]*(\^|>=)[[:space:]]*'"$developmentVersion"'/pragma solidity >0.0.1/' "$f"
```